Maddori.Apple/Maddori.Apple/Global/UIComponent/CollectionView/MemberCollectionView/MemberCollectionView.swift

```diff
@@ -38,7 +38,7 @@ final class MemberCollectionView: UIView {
         let flowLayout = UICollectionViewFlowLayout()
         flowLayout.scrollDirection = .vertical
         flowLayout.sectionInset = Size.collectionInsets
-        flowLayout.itemSize = CGSize(width: Size.cellWidth, height: Size.cellHeight)
+        flowLayout.itemSize = CGSize(width: UIScreen.main.bounds.width - SizeLiteral.leadingTrailingPadding * 4, height: 52)
```

Comment:
p1;
Container 안쪽은 간격이 19 입니다!
```suggestion
        flowLayout.itemSize = CGSize(width: UIScreen.main.bounds.width - SizeLiteral.leadingTrailingPadding * 2 - 38, height: 52)
```
으로 수정 부탁드려요!

Reply:
[7678242](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/347/commits/7678242f51dbefa6a6cd0f1c3829f34662f334d5)

수정완료!